demo/actions.py

```diff
@@ -426,8 +426,13 @@ def run(self,
             ) -> List['Event']:
 
         intent_ranking = tracker.latest_message.get('intent_ranking', [])
+        diff_intent_confidence = intent_ranking[0].get("confidence") - intent_ranking[1].get("confidence")
+        if len(intent_ranking) > 1 and diff_intent_confidence < 0.2:
```

Comment:
the check `len(intent_ranking) > 1` is a bit late since you do `intent_ranking[1].get("confidence")` in the line before

Reply:
good point 🤦‍♀️